gis/gccview/README.md

```diff
@@ -0,0 +1,92 @@
+# GCCVIEW pipeline
+
+## Overview
+
+The GCC (Geospatial Competency Centre) offers a number of spatial products on their arcgis rest api. They host 26 map servers and has over [300 layers](https://github.com/CityofToronto/bdit_data-sources/blob/gcc_view/gis/gccview/available_layers.md) from different divisions within the City of Toronto. To ensure the geospatial layers we use are the most up to date at a regular basis, we are creating a data pipeline to automate the process of retrieving layers from gccview to our postgresql database and conflate street networks through sharedstreets.
+
+Since we can't access the GCC rest API outside the City's firewall, the pipeline has to run in the on-premise server (Morbius / Bancroft) environment, using Airflow 2.1.4.
```

Comment:
since there are layers that can't be access from the GCC's external REST API.

Reply:
modified